### DISCUSSION TITLE
site Article frontend submit set default language

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#32512] [General]
site Article frontend submit set default language

back-end language English
frontend language French

We want create menu item to set to use a category and its subcategories and several other categories, but not all categories.

We want to let users submit articles from frontend only in French language, is it possible?

**We don't want to display flags, because all content is in French** 
![1](https://user-images.githubusercontent.com/76512236/109127672-5b25dc00-7757-11eb-8aab-8149a7489cac.png)

The problem is we get these options: All, English, French. And default is shown as All.

We want to see default language French as in this picture, but we get All.
![Untitled](https://user-images.githubusercontent.com/76512236/109127831-85779980-7757-11eb-83b4-f4dbc05c67bd.png)

Maybe someone knows where is this code ? In what file ? What line?
(What code makes default All, or French or English?)

new j3.24 site, default Protostar template

**Comment by ghost** [comment]
If its technically possible to then it would be really helpful for anyone looking at this show an example how to build this.

**Comment by ot2sen** [comment]
It sounds like you are building a multilingual frontend setup with 1 language. Why?

**Reply by ghost** [reply]
We want
back-end language English
frontend language French

It is 1 French language for frontend, so we don't think we need building multilingual frontend setup

is there better option to do this in j?

**Reply by infograf768** [reply]
This would NOT be a multilingual site. It would be a monolingual site where French UI is used for the site part as default language.
French and English can be used in admin, depending on user's parameters.
Therefore one should not use the language switcher as well as the languagefilter plugin.
